Sources/BrowserServicesKit/Email/EmailKeychainManager.swift

```diff
@@ -127,6 +186,12 @@ private extension EmailKeychainManager {
         deleteItem(forField: .token)
         deleteItem(forField: .alias)
     }
+
+    static func deleteWaitlistState() {
```

Comment:
1. Shouldn't this be part of `deleteAll()` ?
2. I see it only used from debug screen - in case we have implementation that is "dead" on production it would be good to have clear separation of concerns, at least through extensions.

Reply:
`deleteAll` should be renamed, it's used to log out whereas the waitlist state line is only used by the debug VC as you mentioned. I'll try make this more clear. :+1:

Reply:
Renamed `deleteAll` to `deleteAuthenticationState` for clarity, and have moved the functions that are only used for debugging into a demarcated extension at the bottom of the file.